sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -896,35 +896,61 @@ protected override async Task ValidateProcessingPreconditions(CancellationToken
         /// <param name="sequenceNumber">The sequence number to associate with the checkpoint, indicating that a processor should begin reading from the next event in the stream.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> instance to signal a request to cancel the operation.</param>
         ///
-        [EditorBrowsable(EditorBrowsableState.Never)]
         protected override Task UpdateCheckpointAsync(string partitionId,
                                                       long offset,
                                                       long? sequenceNumber,
-                                                      CancellationToken cancellationToken) => UpdateCheckpointAsync(partitionId, new CheckpointPosition(sequenceNumber ?? long.MinValue, offset), cancellationToken);
+                                                      CancellationToken cancellationToken)
+        {
+            if (sequenceNumber.HasValue)
+            {
+                return UpdateCheckpointAsync(partitionId, new CheckpointPosition(sequenceNumber.Value), cancellationToken);
```

Comment:
Not sure that we want to do this.  We'd be dropping the `offset` passed on the floor, no?   Maybe we just do the checkpoint store call in all cases?

Reply:
Yeah I can pull this - I was just trying to align as closely as possible to the way I did it at first which was to just call the new overload, but I think since we're not hiding this anymore + it wouldn't call the new overload in all cases it would probably make more sense to not call the new overload at all.